### PR TITLE
fix compile error on Windows.

### PR DESCRIPTION
**Pull request by FantasyDev-ux**

### CODE REVIEW

**Comment by FantasyDev-ux** [comment]
I fixed the errors while compiling the project on Windows.

**Comment by adamfeuer** [comment]
@FantasyDev-ux I get this error when doing a `cargo build` on Linux:

```
warning: `lsl-sys` (build script) generated 4 warnings
error: failed to run custom build command for `lsl-sys v0.1.0 (/home/adam/work/starcat/engineering/software/hackeeg-client-rust/lsl-sys)`

Caused by:
  process didn't exit successfully: `/home/adam/work/starcat/engineering/software/hackeeg-client-rust/target/debug/build/lsl-sys-602a163d38361b2c/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-changed=wrapper.h
  cargo:rustc-link-search=/home/adam/work/starcat/engineering/software/hackeeg-client-rust/target/debug/build/lsl-sys-fe123d61b6b74dbc/out/liblsl-1.13.0-b14/build
  cargo:rustc-link-lib=static=lsl-static
  cargo:rustc-link-lib=stdc++

  --- stderr
  Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }
```
Did you see this error? Do you know how to fix it?

**Review by adamfeuer — COMMENTED** [review]
resolving conflicts with latest master (that has fixed build)

File: lsl-sys/lib/lsl-static.lib
Comment (adamfeuer) [inline lsl-sys/lib/lsl-static.lib:1]: This can't be checked in. It needs to be built from source.

**Comment by adamfeuer** [comment]
Updated this code and merged it via https://github.com/starcat-io/hackeeg-client-rust/pull/12

**Pull request merged**